config/consensus.go

```diff
@@ -1298,6 +1298,18 @@ func initConsensusProtocols() {
 	vAlpha5.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
 	Consensus[protocol.ConsensusVAlpha5] = vAlpha5
 	vAlpha4.ApprovedUpgrades[protocol.ConsensusVAlpha5] = 10000
+
+	vAlpha6 := vAlpha5
+	vAlpha6.ApprovedUpgrades = map[protocol.ConsensusVersion]uint64{}
+	Consensus[protocol.ConsensusVAlpha6] = vAlpha6
+	vAlpha5.ApprovedUpgrades[protocol.ConsensusVAlpha6] = 10000
```

Comment:
Shorten this down to 1000 or 5000?

Reply:
you have to modify other params to make is less than 10k

Reply:
Oh, so we need 10K voting, 10k upgrade vAlpha6, 10k voting, 10k upgrade vAlpha7, i.e. we need 40K rounds to complete before we can test?